code/game/gamemodes/handofgod/powers.dm

```diff
@@ -0,0 +1,303 @@
+	else
+		src << "You are unaligned, and thus do not have followers"
+		return
+
+	var/datum/mind/choice = input("Choose a follower","Jump to Follower") as null|anything in following
+	if(choice && choice.current)
+		Move(get_turf(choice.current))
+
+
+/mob/camera/god/verb/newprophet()
+	set category = "Deity"
+	set name = "Appoint Prophet (100)"
+	set desc = "Appoint one of your followers as your Prophet, who can hear your words"
+
+	var/list/following = list()
+
+	if(!ability_cost(100))
+		return
+	if(side == "red")
+		var/datum/mind/old_proph = locate() in ticker.mode.red_deity_prophets
+		if(old_proph && old_proph.current && old_proph.current.stat != DEAD)
+			src << "You can only have one prophet alive at a time."
+			return
+		else
+			following = ticker.mode.red_deity_followers
+	else if(side == "blue")
+		var/datum/mind/old_proph = locate() in ticker.mode.blue_deity_prophets
+		if(old_proph && old_proph.current && old_proph.current.stat != DEAD)
+			src << "You can only have one prophet alive at a time."
+			return
+		else
+			following = ticker.mode.blue_deity_followers
+
+	else
+		src << "You are unalligned, and thus do not have prophets"
+		return
+
+	var/datum/mind/choice = input("Choose a follower to make into your prophet","Prophet Uplifting") as null|anything in following
+	if(choice && choice.current && choice.current.stat != DEAD)
+		src << "You choose [choice.current] as your prophet."
+		choice.make_Handofgod_prophet(side)
+
+
+		//Prophet gear
+		var/mob/living/carbon/human/H = choice.current
+		var/popehat = null
+		var/popestick = null
+
+		switch(side)
+			if("red")
+				popehat = /obj/item/clothing/head/helmet/plate/crusader/prophet/red
+				popestick = /obj/item/weapon/godstaff/red
+			if("blue")
+				popehat = /obj/item/clothing/head/helmet/plate/crusader/prophet/blue
+				popestick = /obj/item/weapon/godstaff/blue
+
+		if(popehat)
+			var/obj/item/clothing/head/helmet/plate/crusader/prophet/P = new popehat()
+			P.assign_deity(src)
+
+			H.unEquip(H.head)
+			H << "<span class='boldnotice'>A powerful hat has been bestowed upon your head, you will need to wear this to speak with your god...</span>"
+			H.equip_to_slot_or_del(P,slot_head)
+
+		if(popestick)
+			var/obj/item/weapon/godstaff/G = new popestick()
+			G.god = src
+			var/success = ""
+			if(!H.put_in_any_hand_if_possible(G, qdel_on_fail = 0, disable_warning = 1, redraw_mob = 1))
+				if(!H.equip_to_slot_if_possible(G,slot_in_backpack,0,1,1))
+					G.loc = get_turf(H)
+					success = "It is on the floor..."
+				else
+					success = "It is in your backpack..."
+			else
+				success = "It is in your hands..."
+
+			if(success)
+				H << "<span class='boldnotice'>A powerful staff has been bestowed upon you, you can use this to convert the false god's structures!</span>"
+				H << "<span class=boldnotice'>[success]</span>"
+		//end prophet gear
+
+		add_faith(-100)
+
+
+/mob/camera/god/verb/talk(msg as text)
```

Comment:
i don't see anything that logs this in the say logs, that should be added(should be log_say or something)